src/mumble/main.cpp

```diff
@@ -240,6 +249,10 @@ int main(int argc, char **argv) {
 #endif
 					return 1;
 				}
+			} else if (args.at(i) == QLatin1String("--dump-input-streams")) {
+				 g.bDebugDumpInput = true;
+			} else if (args.at(i) == QLatin1String("--print-echocancel-queue")) {
+				 g.bDebugPrintQueue = true;
```

Comment:
This String that you have changed, is being localized. Thus you'll also have to update the translations file. You can do this by running `scripts/updatetranslations.sh`.
This will do the update and add a commit with it :point_up:

Reply:
Done, translations pass now